src/service/github-allocator-registry/github-allocator-registry.service.ts

```diff
@@ -0,0 +1,147 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { createAppAuth } from '@octokit/auth-app';
+import { Octokit } from '@octokit/core';
+import { ConfigService } from '@nestjs/config';
+import {
+  HealthCheckError,
+  HealthIndicator,
+  HealthIndicatorResult,
+} from '@nestjs/terminus';
+import { LotusApiService } from '../lotus-api/lotus-api.service';
+import { AllocatorRegistry } from './types.github-allocator-registry';
+
+@Injectable()
+export class GitHubAllocatorRegistryService extends HealthIndicator {
+  private readonly logger = new Logger(GitHubAllocatorRegistryService.name);
+  private healthy = true;
+  private octokit: Octokit;
+
+  constructor(
+    private readonly configService: ConfigService,
+    private readonly lotusApiService: LotusApiService,
+  ) {
+    super();
+  }
+
+  public async getHealth(): Promise<HealthIndicatorResult> {
```

Comment:
This function is not being used, as well as `healthy` variable is not being updated anywhere - maybe we dont need healthcheck in this service?

Reply:
Ah, I knew I forgot something. We should set healthy to false when we fail to fetch JSONs from the github. I will fix it.

Reply:
Done: https://github.com/fidlabs/compliance-data-platform/pull/87/commits/84f35a9c550a2564489c0dc8ed035f926f1b7d24